src/bin/nydus-image/core/prefetch.rs

```diff
@@ -168,7 +169,7 @@ impl Prefetch {
         if self.policy == PrefetchPolicy::Fs {
             let mut prefetch_table = RafsV5PrefetchTable::new();
             for i in self.readahead_patterns.values().filter_map(|v| *v) {
-                prefetch_table.add_entry(i as u32);
+                prefetch_table.add_entry(i as u32 + 1);
```

Comment:
why plus one here?

Reply:
Yes. I made a mistake here. will fix it.